node-fs-writefile/cp.ts

```diff
@@ -0,0 +1,11 @@
+import { readFile, writeFile } from 'node:fs/promises';
+
+const [, , source, destination] = process.argv;
+
+try {
+  const data = await readFile(source, 'utf-8');
```

Comment:
By using 'utf-8' here and below you make this code so it can only copy text files. If you simply leave off that argument the code will work for any type of file.